airbyte-webapp/src/locales/en.json

```
@@ -269,17 +269,23 @@
   "sources.cancelled": "Cancelled",
   "sources.incomplete": "Retrying",
   "sources.partialSuccess": "Partial Success",
-  "sources.jobStatus.reset_connection.failed": "Reset Failed ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
-  "sources.jobStatus.reset_connection.running": "Reset Running ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
-  "sources.jobStatus.reset_connection.succeeded": "Reset Succeeded ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
-  "sources.jobStatus.reset_connection.cancelled": "Reset Cancelled ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
-  "sources.jobStatus.reset_connection.partialSuccess": "Reset Partial Success ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
-  "sources.jobStatus.sync.failed": "Sync Failed",
-  "sources.jobStatus.sync.running": "Sync Running",
-  "sources.jobStatus.sync.succeeded": "Sync Succeeded",
-  "sources.jobStatus.sync.cancelled": "Sync Cancelled",
-  "sources.jobStatus.sync.partialSuccess": "Sync Partial Success",
-  "sources.jobStatus.unknown": "Status Unknown",
+  "jobs.jobStatus.reset_connection.failed": "Reset Failed ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
+  "jobs.jobStatus.reset_connection.running": "Reset Running ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
+  "jobs.jobStatus.reset_connection.succeeded": "Reset Succeeded ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
+  "jobs.jobStatus.reset_connection.cancelled": "Reset Cancelled ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
+  "jobs.jobStatus.reset_connection.partialSuccess": "Reset Partial Success ({count, plural, =0 {0 streams} one {# stream} other {# streams}})",
+  "jobs.jobStatus.sync.failed": "Sync Failed",
+  "jobs.jobStatus.sync.running": "Sync Running",
+  "jobs.jobStatus.sync.succeeded": "Sync Succeeded",
+  "jobs.jobStatus.sync.cancelled": "Sync Cancelled",
+  "jobs.jobStatus.sync.partialSuccess": "Sync Partial Success",
+  "jobs.jobStatus.check_connection_source.failed": "Configuration check failed",
+  "jobs.jobStatus.check_connection_source.succeeded": "Configuration check finished",
+  "jobs.jobStatus.check_connection_destination.failed": "Configuration check failed",
+  "jobs.jobStatus.check_connection_destination.succeeded": "Configuration check finished",
```

```suggestion
  "jobs.jobStatus.check_connection_source.failed": "Connection check failed",
  "jobs.jobStatus.check_connection_source.succeeded": "Connection check finished",
  "jobs.jobStatus.check_connection_destination.failed": "Connection check failed",
  "jobs.jobStatus.check_connection_destination.succeeded": "Connection check finished",
```
I think these should be `Connection check ...` instead of `Configuration check ...`. It would be confusing if this said `Configuration check succeeded` but then part of the configuration was wrong, e.g. a wrong pokemon name

---

The reason I tried to avoid "Connection" here was that the we also have "Connections" as a concept in the application, and I found it a bit confusing saying that the "Connection (check) failed", while the user is just setting up connectors. Also not all connectors necessarily do a "connection check", e.g. the pokeapi wouldn't do. So I feel what really happens is that the "Connector configuration is validated", and tried to phrase that.

Regarding your point about "Configuration check succeeded" but the actual configuration being broken, that's exactly why I avoided the word "Succeeded" here, but said "Configuration check finished" in the success case, so it's not being confused with the configuration being actual correct.

Given that thoughts, what would be your preferred way for phrasing here? (I have no strong feelings either way so happy having you make the call on that)

---

Ah okay, I wasn't sure if using `Configuration` instead of `Connection` here was intentional, but I think your justification makes sense, so I think this is fine. I agree that using `finished` instead of `succeeded` for that case makes sense due to the issue I pointed out